### ISSUE TITLE
Amber Interface Broken

### ISSUE THREAD

**Issue by ramirezfranciscof** [MALBECC/lio#285]
I may have detected some problems since #272 (c6eb1cc1378fcc25591cae4ddb9451df6fd6427b) with the amber interface. The watersymbo_verfast test is returning the wrong forces when running born oppenheimer. The previous commit (b5dfd6cf5abfb5ddfe05814040fadba51e6696e0) seems to work fine. Could somebody else check this in your own setup for confirmation? In case of confirmation I might need a hand tracking down the problem @fedepedron .

**Comment by fedepedron** [comment]
I've taken a look at the code and it is weird, since we tested QM/MM forces. The only think I can think of is an error in the energy-weighted density matrix (Pmat_en_wgt) since an error in that one is hard to detect (it usually provides only a small contribution). With verbose > 4 you should get a force log file, if the error lies in ffSG, then it is definitely that matrix.

**Comment by fedepedron** [comment]
Indeed the issue is with Pmat_en_wgt. Will fix ASAP.

**Comment by fedepedron** [comment]
Addressed in #286. Will merge into develop when #281 is merged.

**Comment by fedepedron** [comment]
Also, tests were succeeding because @gonzalodm made the reference (".ok") after #272 xD

**Comment by ramirezfranciscof** [comment]
Well, someone's been a busy bee. Thanks for the quick fix! Will try running with RC untel #281 is merged then.

> Also, tests were succeeding because @gonzalodm made the reference (".ok") after #272 xD

Thank god for the good-old unupdated ehrenfest tests then xD
